listenbrainz/webserver/static/js/src/RecentListens.tsx

```diff
@@ -225,16 +225,37 @@ export default class RecentListens extends React.Component<
   newAlert = (
     type: AlertType,
     title: string,
-    message: string | JSX.Element
+    message: string | JSX.Element,
+    count?: number
   ): void => {
     const newAlert: Alert = {
       id: new Date().getTime(),
       type,
       headline: title,
       message,
+      count,
     };
 
     this.setState((prevState) => {
+      const alertsList = prevState.alerts;
+      for (let i = 0; i < alertsList.length; i += 1) {
+        const item = alertsList[i];
+        if (
+          item.type === newAlert.type &&
+          item.headline.includes(newAlert.headline) &&
```

Comment:
Instead of using `Array.include` here (and in the other components) I suggest `String.startsWith` instead like so:
`item.headline.startsWith(newAlert.headline)`

Reply:
Great suggestion, I'll implement this.